### ISSUE TITLE
Embeddings to predict LISS answers and finally train on PreFer-like task

### ISSUE THREAD

**Issue by tanzir5** [odissei-lifecourse/life-sequencing-dutch#11]
Tasks to do:
1. Finetune the existing LM and network embeddings to predict answer for each of the 31k variables (survey questions) in LISS data.  
2. Use the finetuned model to create synthetic LISS data for the entire population of the Netherlands. 
3. Create fertility data for the entire population by looking at the family (or household) data. We just want to know who in the Netherlands had at least one kid from 2021 to 2023. 
4. Train a traditional ML model (logistic regression or gradient-boosted-random-forest) on the synthetic LISS data to predict real fertility outcome. 
5. Use that model to infer for the test set of PreFer challenge and see if this improves performance.
